### ISSUE TITLE
QA Acceptance test: debian 8 doesn't bootstrap

### ISSUE THREAD

**Issue by ph** [elastic/logstash#5338]
`bootstrap.sh`

```
==> debian-8: Running provisioner: shell...
    debian-8: Running: /var/folders/yl/9hpq_qs528g60cv9trnt5z180000gn/T/vagrant-shell20160520-25625-a6gt3v.sh
==> debian-8: stdin: is not a tty
==> debian-8: /tmp/vagrant-shell: line 3: add-apt-repository: command not found
==> debian-8: Ign http://apt.puppetlabs.com jessie InRelease
==> debian-8: Get:1 http://apt.puppetlabs.com jessie Release.gpg [836 B]
==> debian-8: Get:2 http://security.debian.org jessie/updates InRelease [63.1 kB]
==> debian-8: Get:3 http://apt.puppetlabs.com jessie Release [40.6 kB]
==> debian-8: Get:4 http://apt.puppetlabs.com jessie/PC1 amd64 Packages [7,150 B]
==> debian-8: Get:5 http://security.debian.org jessie/updates/main Sources [136 kB]
==> debian-8: Get:6 http://security.debian.org jessie/updates/contrib Sources [1,439 B]
==> debian-8: Get:7 http://security.debian.org jessie/updates/non-free Sources [14 B]
==> debian-8: Get:8 http://security.debian.org jessie/updates/main amd64 Packages [247 kB]
==> debian-8: Ign http://apt.puppetlabs.com jessie/PC1 Translation-en_US
==> debian-8: Ign http://apt.puppetlabs.com jessie/PC1 Translation-en
==> debian-8: Ign http://cloudfront.debian.net jessie InRelease
==> debian-8: Get:9 http://security.debian.org jessie/updates/contrib amd64 Packages [2,506 B]
==> debian-8: Get:10 http://cloudfront.debian.net jessie Release.gpg [2,373 B]
==> debian-8: Get:11 http://security.debian.org jessie/updates/non-free amd64 Packages [14 B]
==> debian-8: Get:12 http://security.debian.org jessie/updates/contrib Translation-en [1,211 B]
==> debian-8: Get:13 http://security.debian.org jessie/updates/main Translation-en [135 kB]
==> debian-8: Get:14 http://cloudfront.debian.net jessie Release [148 kB]
==> debian-8: Get:15 http://security.debian.org jessie/updates/non-free Translation-en [14 B]
==> debian-8: Get:16 http://cloudfront.debian.net jessie/main Sources [7,059 kB]
==> debian-8: Get:17 http://cloudfront.debian.net jessie/non-free Sources [99.2 kB]
==> debian-8: Get:18 http://cloudfront.debian.net jessie/contrib Sources [50.7 kB]
==> debian-8: Get:19 http://cloudfront.debian.net jessie/main amd64 Packages [6,763 kB]
==> debian-8: Get:20 http://cloudfront.debian.net jessie/non-free amd64 Packages [83.6 kB]
==> debian-8: Get:21 http://cloudfront.debian.net jessie/contrib amd64 Packages [50.1 kB]
==> debian-8: Get:22 http://cloudfront.debian.net jessie/contrib Translation-en [38.5 kB]
==> debian-8: Get:23 http://cloudfront.debian.net jessie/main Translation-en [4,581 kB]
==> debian-8: Get:24 http://cloudfront.debian.net jessie/non-free Translation-en [72.5 kB]
==> debian-8: Fetched 19.6 MB in 14s (1,367 kB/s)
==> debian-8: Reading package lists...
==> debian-8: Reading package lists...
==> debian-8: Building dependency tree...
==> debian-8: Reading state information...
==> debian-8: Package openjdk-8-jdk is not available, but is referred to by another package.
==> debian-8: This may mean that the package is missing, has been obsoleted, or
==> debian-8: is only available from another source
==> debian-8: E
==> debian-8: :
==> debian-8: Package 'openjdk-8-jdk' has no installation candidate
==> debian-8: update-alternatives: error: no alternatives for java
==> debian-8: update-alternatives: error:
==> debian-8: no alternatives for javac
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.``

**Comment by purbon** [comment]
All current vms can bootstrap as tested, see:

```
purbon@slave-524707:~/logstash/qa$ vagrant status
Current machine states:

centos-6                  running (virtualbox)
centos-7                  running (virtualbox)
debian-8                  running (virtualbox)
fedora-22                 running (virtualbox)
oel-6                     running (virtualbox)
oel-7                     running (virtualbox)
opensuse-13               running (virtualbox)
sles-12                   running (virtualbox)
ubuntu-1204               running (virtualbox)
ubuntu-1404               running (virtualbox)
ubuntu-1504               running (virtualbox)

This environment represents multiple VMs. The VMs are all listed
above with their current state. For more information about a specific
VM, run `vagrant status NAME`.
```

**Comment by purbon** [comment]
Fixed in #5346
